app/models/project.rb

```diff
@@ -3,4 +3,6 @@ class Project < ApplicationRecord
 
     has_many :databases
     has_many :collaborators, dependent: :destroy
+
+    has_one_attached :image
```

Comment:
I would suggest using a more specific name like `project_background_image` or `project_image`. If we want to add more images to more things in the future, we might get confused at to what `image` belongs to.

If you use global find (Cmd + Shift + F), you can easily find all `:image`s in all files and replace them.

Reply:
Yea that's also true ;-)